xml/tuning-ceph-tuning.xml

```diff
@@ -409,18 +438,17 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
      a client, there are some client options that are covered in the
      client section.
    </para>
-   <sect2 xml:id="mds-tuning">
+   <sect2>
      <title>MDS Tuning</title>
      <para>
        In filesystems with millions of files, there is some advantage to
        utilizing very low-latency media, such as NVMe, for the cephfs metadata pool.
      </para>
      <para>
-       Utilizing the <command>ceph daemon perf dump</command> command (for example,
-       <command>ceph daemon mds.node3 perf dump</command>), there is a significant
+       Utlizing the <command>ceph-daemon perf dump</command> command, there is a significant
        amount of data that can be examined for the &ceph; Metadata Servers.
-       It should be noted that the MDS performance counters only apply to metadata
-       operations. The actual I/O path is from clients straight to OSDs.
+       It should be noted that the MDS perf counters only apply to metadata
```

Comment:
Why the change from the full word to an abbreviation? I would favour the full word.

Reply:
They're called perf counters: https://docs.ceph.com/docs/master/dev/perf_counters/